src/plugins/mer/mervirtualboxmanager.h

```diff
@@ -56,7 +56,7 @@ class VirtualMachineInfo
     bool headless;
 };
 
-class MerVirtualBoxManager : public QObject
+class Q_DECL_EXPORT MerVirtualBoxManager : public QObject
```

Comment:
I would prefer keeping this internal and expose the added function through MerEmulatorDevice instead.